### ISSUE TITLE
browser only?

### ISSUE THREAD

**Issue by coderofsalvation** [eastolfi/MongoPortable#3]
```
$ node --version
v6.9.1
$ node example-in-readme.js
window not found -> not a browser environment
/tmp/test.js:5
var db = new MongoPortable("TEST");
         ^

TypeError: MongoPortable is not a constructor
    at Object.<anonymous> (/tmp/test.js:5:10)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

**Comment by coderofsalvation** [comment]
aha, the example code was missing this: 

https://github.com/EastolfiWebDev/MongoPortable/pull/4

**Comment by eastolfi** [comment]
Closed as PR: #4 is merged
